cadc-test-vos/src/main/java/ca/nrc/cadc/conformance/vos/GetContainerNodeTest.java

```diff
@@ -176,6 +177,295 @@ public void getContainerNode()
         }
     }
 
+
```

Comment:
Can this test be used as a conformance test by someone that does not implement sorted output? I think probably not and did not see anything to make it optional. However, that is probably a major problem with this test suite so I'm not saying fix it -- just that this test design constraint was probably not taken into account.